context/app/static/js/components/detailPage/files/DataProducts/FileSize.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import prettyBytes from 'pretty-bytes';
+import Typography, { TypographyProps } from '@mui/material/Typography';
+
+type FileSizeProps = {
+  size: number;
+  variant?: TypographyProps['variant'];
+};
+
+export function FileSize({ size, variant = 'body2' }: FileSizeProps) {
+  return (
+    <Typography component="p" variant={variant} color="#00000099">
```

Comment:
Did this color come from the design? If this is purposeful, we should introduce it to the theme.

Reply:
This color _is_ in the design, but we don't have it in our palette yet. 

It probably won't hurt to add it to the theme; I was planning to do a full sweep to update it to match Tiffany's palette for Material Design 3 in Figma and make our theme extensions typesafe, but adding an extra palette item wouldn't significantly affect the scope of that task.